PhysicsTools/PatAlgos/python/producersLayer1/jetProducer_cfi.py

```diff
@@ -99,6 +99,47 @@
     resolutions     = dict()
 )
 
+from Configuration.Eras.Modifier_run3_common_cff import run3_common
+run3_common.toModify(_patJets,
+              jetSource = "ak4PFJetsCHS",
+              genJetMatch = "patJetGenJetMatch",
+              genPartonMatch = "patJetPartonMatch",
+              JetFlavourInfoSource = "patJetFlavourAssociation",
+              JetPartonMapSource = "patJetFlavourAssociationLegacy",
+              jetCorrFactorsSource = ["patJetCorrFactors"],
+              addBTagInfo          = True,   ## master switch
+              addDiscriminators    = True,   ## addition btag disc.
+              discriminatorSources = ["pfJetBProbabilityBJetTags",
+                                      "pfJetProbabilityBJetTags",
+                                      "pfTrackCountingHighEffBJetTags",
+                                      # DeepFlavour
```

Comment:
This is confusing to me - DeepCSV is not DeepFlavour, is it? Why are the DeepFlavour tags commented below?

Reply:
This part is a direct copy of original L. 52 to L. 63.